translate/ir.py

```diff
@@ -0,0 +1,110 @@
+from dataclasses import dataclass
+import typing
+import ast
+
+
+@dataclass
+class ImportedSymbol:
+    name: str
```

Comment:
Not sure what to do here.  `pep8ify` translates this to:
```python
class ImportedSymbol:
    name:
        str
    alias:
        str
```

But then I get a syntax error if I try to run it:
```
Traceback (most recent call last):
  File "/home/marler8997/git/autowrap/translate/test_ir.py", line 3, in <module>
    import ir
  File "/home/marler8997/git/autowrap/translate/ir.py", line 8
    name:
        ^
SyntaxError: invalid syntax
```

So I'm having to manually fix it to be on one line after I run `pep8ify`.

Reply:
Argh, tools.